### PR TITLE
Update roles

### PR DESCRIPTION
**Pull request by ajnelson-nist**
# Committer Notes

This PR updates the project metadata to reflect role updates.

### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/usnistgov/oss-maven/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/oss-maven/pulls) for the same update/change?
- [x] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [x] Do all automated CI/CD checks pass?

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your core changes, as applicable?
- [x] Have you included examples of how to use your new feature(s)?
- [x] Have you updated all website and readme documentation affected by the changes you made?

### CODE REVIEW

**Comment by ajnelson-nist** [comment]
This PR is expected to pass CI before merging, but due to a currently pending account matter, will fail CI after merging.

**Comment by ajnelson-nist** [comment]
@david-waltermire , can you please review this and update as you see appropriate?

File: pom.xml
Comment (david-waltermire) [inline pom.xml]: I won't be maintaining this anymore.
```suggestion
```

**Review by david-waltermire — CHANGES_REQUESTED** [review]
I should be removed as "maintainer".

**Comment by ajnelson-nist** [comment]
> I should be removed as "maintainer".

Change accepted with the GitHub interface.

**Review by david-waltermire — APPROVED** [review]
LGTM!

**Pull request merged**
